build_artifacts/v0/v0.13/v0.13.0/RELEASE.md

```diff
@@ -0,0 +1,32 @@
+# Release notes: 0.13.0
+
```

Comment:
Can you manually leave a note that we upgraded Python from 3.8 to 3.11

Reply:
Updated